### ISSUE TITLE
1d plot

### ISSUE THREAD

**Issue by dmargarone** [fbpic/fbpic#548]
Hi Guys !

Thanks for the nice code. I am enjoying using it !!
I want to make a 1d line out of the electric field profile. How I can do this using OpenPMD. 

Thanks.
DM

**Comment by dmargarone** [comment]
Hi guys,

any clue .... 

I also tried E1d = ts.get_field(field='E', coord='x', slicing=[0., 0.], slicing_dir=['x', 'y']), but it didn't worked.

waiting for reply !

**Comment by lauridsj** [comment]
Hi @dmargarone,

you can do an 1D slice using the slicing feature of openPMD-viewer, like this:

`E1d, meta = ts.get_field("E", "z", iteration=20, slice_across="r", slice_relative_position=0.)`

You can also find more options for openPMD in the relevant docstrings (if you use jupyter, they are available by simply typing `ts.get_field?`).

Hope this helps!
Cheers, Laurids

**Comment by dmargarone** [comment]
Hi Laurids,

Thanks for your help ... really appreciable. However, it is not completely solved yet. Please, could you see the attached pdf, which shows what I get with the command you suggested and what I am looking for. 

Thanks. 
[1D_llineout.pdf](https://github.com/fbpic/fbpic/files/6912001/1D_llineout.pdf)

**Comment by lauridsj** [comment]
Hi,

I'm not completely sure where the problem lies - the command you used should produce only one 1D lineout. What did you use to plot this? With pyplot, you can do
`plt.plot(meta.z, E1d)`
and it should give you your 1D plot.

If that does not work, can you maybe post your entire code? I'll might be able to spot the problem then...

**Comment by dmargarone** [comment]
Hi, Laurids

Thank you so much for your quick reply. Please, could you check the attached files: (1) simulation file (lwfa_script.py), and (2) Ex_1d_test.py (post-processing file) to get the 1d profile.

Thank you so much.

[1D_ploting_problem.zip](https://github.com/fbpic/fbpic/files/6913722/1D_ploting_problem.zip)

**Comment by lauridsj** [comment]
Hi,

I tried your code and it gave me this nice plot:
![grafik](https://user-images.githubusercontent.com/9991635/127828476-ac52257f-48be-4f7f-b903-138333f13286.png)

I'm not sure why it does not work for you, but I noticed that you seem to be using an old version of openPMD_viewer, according to your import statements (`opmd_viewer` instead of `openpmd_viewer`). Maybe try updating openPMD_viewer with pip and switching to the new import statement (i.e. `from openpmd_viewer import OpenPMDTimeSeries` etc.)?

Cheers,
Laurids
